sdk/maps/azure-maps-search/azure/maps/search/_search_client.py

```diff
@@ -65,1050 +45,387 @@ def __init__(
+         to see the available Views. Default value is None.
+        :paramtype view: str
+        :return: GeocodingResponse
+        :rtype: ~azure.maps.search.models.GeocodingResponse
+        :raises ~azure.core.exceptions.HttpResponseError:
         """
-        batch_items = [{"query": f"?query={query}"} for query in search_queries] if search_queries else []
-
-        poller = self._search_client.begin_search_address_batch(
-            batch_request={"batch_items": batch_items},
+        result = self._search_client.get_reverse_geocoding(
+            coordinates=coordinates,
+            result_types=result_types,
+            view=view,
             **kwargs
         )
-
-        return poller
+        return result
 
     @distributed_trace
-    def reverse_search_address_batch(
-        self,
-        search_queries: List[str],
-        **kwargs: Any
-    ) -> ReverseSearchAddressBatchProcessResult:
-        """**Search Address Reverse Batch API**
-
-        The Search Address Batch API sends batches of queries to `Search Address Reverse API`.
-        You can call Search Address Reverse Batch API to run either asynchronously (async) or
-        synchronously (sync). The async API allows caller to batch up to **10,000** queries and sync
-        API up to **100** queries.
-
-        :param search_queries:
-            The list of search fuzzy queries/requests to process.
-            The list can contain a max of 10,000 queries and must contain at least 1 query.
-        :type search_queries:
-            List[str]
-        :return:
-            The results of search batch request.
-        :rtype:
-            ~azure.maps.search.models.ReverseSearchAddressBatchProcessResult
-        :raises:
-            ~azure.core.exceptions.HttpResponseError:
+    def get_reverse_geocoding_batch(
+            self,
+            reverse_geocoding_batch_request_body: Union[_models.ReverseGeocodingBatchRequestBody, IO[bytes]],
+            **kwargs: Any
+    ) -> _models.GeocodingBatchResponse:
+        """Use to send a batch of queries to the `Reverse Geocoding
+        </rest/api/maps/search/get-reverse-geocoding>`_ API in a single request.
+
+        The ``Get Reverse Geocoding Batch`` API is an HTTP ``POST`` request that sends batches of up to
+        **100** queries to `Reverse Geocoding </rest/api/maps/search/get-reverse-geocoding>`_ API using
+        a single request.
+
+        Submit Synchronous Batch Request
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+        The Synchronous API is recommended for lightweight batch requests. When the service receives a
+        request, it will respond as soon as the batch items are calculated and there will be no
+        possibility to retrieve the results later. The Synchronous API will return a timeout error (a
+        408 response) if the request takes longer than 60 seconds. The number of batch items is limited
+        to **100** for this API.
+
+        .. code-block::
```

Comment:
We should remove the code block in docstring since the docstring will be rendered to on-line document. We should keep the API description simple.